src/irrlicht_changes/CGUITTFont.cpp

```diff
@@ -246,54 +300,22 @@ shadow_offset(0), shadow_alpha(0), fallback(0)
 	setInvisibleCharacters(L" ");
 }
 
-bool CGUITTFont::load(const io::path& filename, const u32 size, const bool antialias, const bool transparency)
+bool CGUITTFont::load(SGUITTFace *face, const u32 size, const bool antialias, const bool transparency)
 {
 	// Some sanity checks.
 	if (!Driver) return false;
 	if (size == 0) return false;
-	if (filename.empty()) return false;
+	if (!face) return false;
 
 	this->size = size;
-	this->filename = filename;
 
 	// Update the font loading flags when the font is first loaded.
 	this->use_monochrome = !antialias;
 	this->use_transparency = transparency;
 	update_load_flags();
 
-	infostream << "CGUITTFont: Creating new font: " << filename.c_str() << " "
-		<< size << "pt " << (antialias ? "+antialias " : "-antialias ")
-		<< (transparency ? "+transparency" : "-transparency") << std::endl;
```

Comment:
no more logging?

Reply:
Yes, this logging isn't very useful. There already is a terser log message in `fontengine.cpp`:

```cpp
infostream << "Creating new font: " << font_path.c_str() << " " << size << "pt" << std::endl;
```